README.md

```diff
@@ -1 +1,67 @@
-# nmpolicy
\ No newline at end of file
+---
+---
+
+# Introduction
+
+An expressions driven declarative API for dynamic network configuration
+
+# Motivation
+
+When networking configuration for a cluster is needed and all the details 
+are common between the nodes in the cluster a NMState yaml configuration is 
+enough.
+
+Problems arise when some of the network configuration details 
+are different between nodes and depend on the current node network state.
+
+For that a different NMState yaml configuration needs to be generated 
+per node and that's not convenient for big clusters and 
+also at scale up scenarios.
+
+The NMPolicy goal is to solve this problem.
+Given a node network state and a network configuration policy 
+(common to the cluster), 
+the NMPolicy tool will generate a node specific desired network state.
+
+# How it works
+
+It's implemented on top of [nmstate](https://nmstate.io/), nmpolicy generates a 
+[nmstate](nmstate.io) desired state as output, given an input of a 
+[policy spec](nmstate.io/nmpolicy/user-guide/102-policy-syntax.html) and a 
```

Comment:
This will work when we merge the PR, at the `README.md` I can not reference stuff locally or with Jekyll primitives since it's the GitHub frontend README and is not interpreted by Jekyll.